spring-cloud-stream-binder-kstream/src/main/java/org/springframework/cloud/stream/binder/kstream/config/KStreamProducerProperties.java

```diff
@@ -16,10 +16,31 @@
 
 package org.springframework.cloud.stream.binder.kstream.config;
 
+import org.springframework.cloud.stream.binder.kafka.properties.KafkaProducerProperties;
+
 /**
  * @author Marius Bogoevici
  * @author Soby Chacko
  */
-public class KStreamProducerProperties extends KStreamCommonProperties {
+public class KStreamProducerProperties extends KafkaProducerProperties {
```

Comment:
Perhaps an abstract class as a base for producer/consumer properties given that they expose the same name properties